src/msg/msg_types.h

```diff
@@ -51,6 +51,7 @@ class entity_name_t {
   static const int TYPE_OSD = CEPH_ENTITY_TYPE_OSD;
   static const int TYPE_CLIENT = CEPH_ENTITY_TYPE_CLIENT;
   static const int TYPE_MGR = CEPH_ENTITY_TYPE_MGR;
+  static const int TYPE_EXPORTER = CEPH_ENTITY_TYPE_EXPORTER;
```

Comment:
what's the motivation for a new entity type? can we just use `TYPE_CLIENT` here?

Reply:
It's no more needed